metrics/metrics.go

```diff
@@ -151,8 +172,22 @@ var (
 		[]string{"url"},
 	)
 
-	// DrandBuildTime emits the timestamp when the binary was built in Unix time.
-	DrandBuildTime = prometheus.NewUntypedFunc(prometheus.UntypedOpts{
+	// dKGStateChangeTimestamp tracks DKG status changes
+	dKGStateChangeTimestamp = prometheus.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
`dkgStateChangeTimestamp` would be nicer, no?

Reply:
right... I had it as exported and then just put the first letter as lowercase since it does not need to be exported.